tools/misc/browse_dataset.py

```diff
@@ -56,9 +62,13 @@ def retrieve_data_cfg(config_path, skip_type, cfg_options):
     while 'dataset' in train_data_cfg and train_data_cfg[
             'type'] != 'MultiImageMixDataset':
         train_data_cfg = train_data_cfg['dataset']
-    train_data_cfg['pipeline'] = [
-        x for x in train_data_cfg.pipeline if x['type'] not in skip_type
-    ]
+
+    if isinstance(train_data_cfg, (list, tuple)):
```

Comment:
(list, tuple) can be simplified as sequence

Reply:
i followed the same "code pattern" as in the concatenated dataset build method and generally in the project but i can use abc Sequence if you prefer.

Reply:
changed